update.js

```diff
@@ -0,0 +1,41 @@
+var fs = require('fs')
+var chalk = require('chalk')
+var resolve = require('path').resolve
+var join = require('path').join
+var cp = require('child_process')
+var os = require('os')
+
+if(fs.existsSync('config.json')) {
```

Comment:
I See this really complicated. The build being dependent on a runtime config File feels so wrong.

Just Walk into all sub folders within apps

Reply:
if i just walk into every app folder this would break build on apps that may be WIP or broken atm which still are in my working dir. this way i can exclude them to the minimum required to drone config.

Reply:
Still .... Im not convinced. Had enough issues already ....